lib/init/parser.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const parser = require('body-parser');
```

Comment:
I understand the bug. `connect-multiparty` does the same thing, and will give `req.body.email` as well.